pkg/oc/admin/migrate/storage/storage.go

```diff
@@ -161,6 +164,9 @@ func NewCmdMigrateAPIStorage(name, fullName string, f *clientcmd.Factory, in io.
 		Long:    internalMigrateStorageLong,
 		Example: fmt.Sprintf(internalMigrateStorageExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
+			apiextensioninstall.Install(legacyscheme.GroupFactoryRegistry, legacyscheme.Registry, legacyscheme.Scheme)
```

Comment:
Brute force. Wondering how kubectl is importing the internal types right now (not everything is ported to versioned types). Looks like there is no single point where everything is imported (anymore).